docs/website/sidebars.js

```diff
@@ -136,6 +136,19 @@ const sidebars = {
         'reference/explainers/how-dlt-works',
         'general-usage/resource',
         'general-usage/source',
+        {
+          type: 'category',
+          label: 'Configuration and secrets',
+           link: {
+            type: 'doc',
+            id: 'general-usage/credentials/index',
+          },
+          items: [
+            'general-usage/credentials/setup',
```

Comment:
Since we're renaming the section, before merging to master, we'd need to set up redirect from the older pages.

Reply:
good spotting!